build.sbt

```diff
@@ -116,19 +118,11 @@ lazy val interopTests = Project(
   )
   .settings(Dependencies.interopTests)
   .settings(commonSettings)
-  .enablePlugins(JavaAgent)
+  .pluginTestingSettings
   .settings(
-    ReflectiveCodeGen.reflectiveGeneratorClass := "akka.grpc.sbt.test.AkkaCompositeCodeGenerator",
-    javaAgents += Dependencies.Agents.jettyAlpnAgent % "test",
-    // needed explicitly as we don't directly depend on the codegen project
-    watchSources ++= (watchSources in codegen).value,
-    // yeah ugly, but otherwise, there's a circular dependency between the project values
-    watchSources ++= (watchSources in ProjectRef(file("."), "sbt-akka-grpc")).value,
-  )
-  .dependsOn(runtime)
-  .enablePlugins(akka.grpc.ReflectiveCodeGen)
-  // needed to be able to override the PB.generate task reliably
-  .disablePlugins(ProtocPlugin)
+    ReflectiveCodeGen.generatedLanguages := "Scala, Java",
```

Comment:
this seems a bit weird that its a string like that...? Why not a specific type that I think we have already?

Reply:
It's because of the setup where we cannot directly reference settings from the sbt plugin but have to mimic it as far as possible.

Reply:
So the idea is to keep this mimic'd structure as light as possible being just enough to support running the test project inside the same project.

Reply:
Ah I see... okey!